packages/Webkul/Shop/src/Http/Resources/ProductResource.php

```diff
@@ -50,7 +50,7 @@ public function toArray($request)
             'price_html'  => $productTypeInstance->getPriceHtml(),
             'ratings'     => [
                 'average' => $this->reviewHelper->getAverageRating($this),
-                'total'   => $this->reviewHelper->getTotalRating($this),
+                'total'   => $this->reviewHelper->getTotalFeedback($this),
             ],
```

Comment:
~~~php
...

'ratings'     => [
    'average' => $this->reviewHelper->getAverageRatings($this),
    'total'   => $this->reviewHelper->getTotalRatings($this),
],

'reviews'     => [
    'total'   => $this->reviewHelper->getTotalReviews($this),
],

...
~~~